internal/core/src/segcore/storagev2translator/GroupChunkTranslator.h

```diff
@@ -78,7 +78,17 @@ class GroupChunkTranslator
     int64_t
     cells_storage_bytes(
         const std::vector<milvus::cachinglayer::cid_t>& cids) const override {
-        return 0;
+        constexpr int64_t MIN_STORAGE_BYTES = 1 * 1024 * 1024;
+        int64_t total_size = 0;
+        for (auto cid : cids) {
+            auto [file_idx, row_group_idx] = get_file_and_row_group_index(cid);
+            auto& row_group_meta =
+                row_group_meta_list_[file_idx].Get(row_group_idx);
+            total_size +=
+                std::max(static_cast<int64_t>(row_group_meta.memory_size()),
```

Comment:
if 2 cells are consecutive, and both are less than 1MB and sum is also less than 1MB, we should charge 1MB instead of 2MB? because we'll make only a single io